### DISCUSSION TITLE
Message when trying to connect to my mysql 3.51 CLIENT_SECURE_CONNECTION is required

### DISCUSSION THREAD

**Discussion by imbesi** [dbeaver/dbeaver#11466] [Q&A] [answered]
I try to connect to my MySql 3.51 server based on a Linux server I have in my office, connection working perfectly for years on old Navicat connection, I receive this message "CLIENT_SECURE_CONNECTION is required".
I set my MySql to version OLD.
Please can you tell me how to solve problem?
Thanks
Domenico

#### System information: 
- Operating system (distribution) and version
- DBeaver version
- Java version 
- Additional extensions

#### Connection specification:
- Database name and version
- Driver name
- Do you use tunnels or proxies (SSH, SOCKS, etc)?

**Comment by kseniiaguzeeva** [comment, marked as answer]
Please try to use SSL:
Fill necessary fields and enable Verify server certificate and Allow public key retrieval there.
![Screenshot 2021-01-18 at 18 00 35](https://user-images.githubusercontent.com/51405061/104931278-3e0f2800-59b7-11eb-8493-41e84e010691.png)
